vue-components/src/components/Checkbox.vue

```diff
@@ -60,8 +60,8 @@ $label: '.wikit-checkbox__label';
 		align-items: center;
 
 		&::before {
-			width: $wikit-Checkbox-input-size;
```

Comment:
I removed the `width` and `height` properties because I understand they're being overridden by the min properties, but please correct me if I'm wrong.

Reply:
Tested. It works as expected.